INFO (theano.gof.compilelock): Waiting for existing lock by process '43392' (I am process '43093')
INFO (theano.gof.compilelock): To manually release the lock, delete /n/home02/nmudur/.theano/compiledir_Linux-3.10-el7.x86_64-x86_64-with-centos-7.6.1810-Core-x86_64-2.7.16-64/lock_dir
INFO (theano.gof.compilelock): Waiting for existing lock by process '43392' (I am process '43093')
INFO (theano.gof.compilelock): To manually release the lock, delete /n/home02/nmudur/.theano/compiledir_Linux-3.10-el7.x86_64-x86_64-with-centos-7.6.1810-Core-x86_64-2.7.16-64/lock_dir
INFO (theano.gof.compilelock): Waiting for existing lock by process '43392' (I am process '43093')
INFO (theano.gof.compilelock): To manually release the lock, delete /n/home02/nmudur/.theano/compiledir_Linux-3.10-el7.x86_64-x86_64-with-centos-7.6.1810-Core-x86_64-2.7.16-64/lock_dir
Using cuDNN version 7005 on context None
Mapped name None to device cuda: Tesla V100-PCIE-32GB (0000:2F:00.0)
/n/home02/nmudur/.conda/envs/theano_env/lib/python2.7/site-packages/theano/tensor/subtensor.py:2339: FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
  out[0][inputs[2:]] = inputs[1]
